### ISSUE TITLE
error: The class 'AfterLayoutMixin' can't be used as a mixin because it extends a class other than Object

### ISSUE THREAD

**Issue by shashkiranr** [fluttercommunity/flutter_after_layout#3]
Unable to use this package.

```
Flutter 0.10.2-pre.53 • channel master • https://github.com/flutter/flutter.git
Framework • revision 549e8e07c6 (4 days ago) • 2018-10-24 14:41:16 -0700
Engine • revision 2586e94122
Tools • Dart 2.1.0-dev.7.1.flutter-45f9462398
```

**Comment by g-balas** [comment]
Same here.

**Comment by zoechi** [comment]
Changing `class` to `mixin` should fix that.

**Comment by g-balas** [comment]
> Changing `class` to `mixin` should fix that.

Can you be a bit more specific please ?

**Comment by zoechi** [comment]
Replace `abstract class` by `mixin` in https://github.com/fluttercommunity/flutter_after_layout/blob/master/lib/after_layout.dart#L5
You probably need `dev` or `master` channel Flutter for `mixin` to work

**Comment by bagintz** [comment]
Adding an 'analysis_options.yaml' in the root directory also helped this issue until a couple of days ago, not sure what changed

```
analyzer:
  language:
    enableSuperMixins: true
```

**Comment by zoechi** [comment]
`enableSuperMixins` was an experiment from pre 2.0.0 and was kept alive for Flutter.
Recently the new `mixin` feature became available in Dart and the experiment was removed.
The new `mixin` feature will probable start working after the next Flutter engine roll.

**Comment by shashkiranr** [comment]
> Replace `abstract class` by `mixin` in https://github.com/fluttercommunity/flutter_after_layout/blob/master/lib/after_layout.dart#L5
> You probably need `dev` or `master` channel Flutter for `mixin` to work

This Worked for me . so is this a workaround or the final solution?

**Comment by slightfoot** [comment]
Issue resolved by #4
